frontend/components/AgentForms/MemeooorrAgentForm/MemeooorrAgentForm.tsx

```diff
@@ -137,7 +157,11 @@ export const MemeooorrAgentForm = ({
 
       {/* X account tokens */}
       <Divider style={{ margin: '8px 0' }} />
-      <XAccountApiTokens />
+      <XAccountApiTokens
+        showTokensRequiredMessage={
+          !isMemeooorrFieldUpdateCompleted && variant !== 'borderless'
```

Comment:
well a bit weird, that business logic depends on styling. is it related to editing/viewing mode? maybe can use isFormEnabled?